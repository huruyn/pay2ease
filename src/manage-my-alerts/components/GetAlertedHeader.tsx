import { Col, Row } from 'react-bootstrap';

export default function GetAlertedHeader() {
  return (
    <Row
      style={{
        backgroundColor: '#EFEFEF',
        borderBottom: '1px solid grey',
      }}
    >
      <Col md={8}>
        <p style={{ marginTop: 12 }}>GET ALERTED WHEN...</p>
      </Col>
      <Col md={4}>
        <p style={{ marginTop: 12 }}>SEND ALERT TO...</p>
      </Col>
    </Row>
  );
}
