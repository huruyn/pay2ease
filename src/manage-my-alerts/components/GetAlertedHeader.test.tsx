import React from 'react';
import GetAlertedHeader from './GetAlertedHeader';

const ReactTestRenderer = require('react-test-renderer');

describe('GetAlertedHeader Component', () => {
  it('matches the snapshot', () => {
    const tree = ReactTestRenderer.create(
      <GetAlertedHeader />,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
