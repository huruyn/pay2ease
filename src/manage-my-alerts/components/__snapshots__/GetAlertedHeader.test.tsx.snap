// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GetAlertedHeader Component matches the snapshot 1`] = `
<div
  className="row"
  style={
    Object {
      "backgroundColor": "#EFEFEF",
      "borderBottom": "1px solid grey",
    }
  }
>
  <div
    className="col-md-8"
  >
    <p
      style={
        Object {
          "marginTop": 12,
        }
      }
    >
      GET ALERTED WHEN...
    </p>
  </div>
  <div
    className="col-md-4"
  >
    <p
      style={
        Object {
          "marginTop": 12,
        }
      }
    >
      SEND ALERT TO...
    </p>
  </div>
</div>
`;
